lms/djangoapps/philu_api/helpers.py

```diff
@@ -35,12 +35,21 @@ def get_social_sharing_urls(course_url, meta_tags):
         'twitter': _compile_social_sharing_url(SOCIAL_SHARING_URLS['twitter']['url'], course_share_url,
                                                SOCIAL_SHARING_URLS['twitter']['url_param'],
                                                SOCIAL_SHARING_URLS['twitter']['utm_source'],
-                                               text=TWITTER_MESSAGE_FORMAT.format(meta_tags['title']))
+                                               text=TWITTER_MESSAGE_FORMAT.format(meta_tags['title'])),
+
+        'email': _compile_social_sharing_url(SOCIAL_SHARING_URLS['email']['url'], course_share_url,
+                                             SOCIAL_SHARING_URLS['email']['url_param'],
+                                             SOCIAL_SHARING_URLS['email']['utm_source']),
+
     }
 
 
 def _compile_social_sharing_url(share_url, course_url, url_param, utm_source, text=None):
     course_url_with_utm = add_or_replace_parameter(course_url, 'utm_source', utm_source)
+
+    if not share_url:
```

Comment:
Though the variables names are describing the sanity check. But still, let's add a comment to mention why we need to add this sanity here.